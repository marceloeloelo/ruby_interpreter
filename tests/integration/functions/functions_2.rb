def method_1
  1
end

def method_2 x, y
  x * y
end

def method_3(x, y, z)
  puts(x + y)
  puts("entre aca")
  z = x
  z
end

puts(method_1)
puts(method_2(1 + 2, "hola"))

a = method_2(2, 2+1)
puts(a)
puts(a + 1)

puts(b = method_2(3, 3))
puts(b)

b = "hola "
c = method_3(b, "chau", 10)
puts(c)
