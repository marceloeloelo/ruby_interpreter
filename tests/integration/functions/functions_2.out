1
holaholahola
6
7
9
9
hola chau
entre aca
hola 
